gui.c

```diff
@@ -28,6 +26,7 @@ enum
 
 GtkBuilder *builder;
 
+char *DOWNLOADS_PATH = "/home/compro/Downloads/";
```

Comment:
Are you sure this should be here?

Reply:
The better way is to have a settings/configuration context object passed everywhere but that will need more work on a larger scale (i.e. we can have a settings panel to change these things). We will need to get there step by step. For now, its okay.
I also want your opinion on whether it would be better to use C++ or C(or Rust). Please elaborate. I ask this because I don't have much experience with Gtk.

Reply:
I would suggest continuing with C (never using C++, I dont know about gtk in rust at all) for the moment as good amount of work is already done. If there is any issue with the usage of the same, you can open an issue about that and tag me, will look into it.

Reply:
As for the line itself, replace 'compro' with 'user'

Reply:
Totally missed that.